google/cloud/sql/connector/instance_connection_manager.py

```diff
@@ -250,6 +251,10 @@ def __init__(
         self._keys = asyncio.wrap_future(keys, loop=self._loop)
         self._auth_init()
 
+        self._refresh_rate_limiter = AsyncRateLimiter(
+            burst_size=1, interval=60, queue_size=1, loop=self._loop
```

Comment:
I think we probably want to limit this at 2 per 60s (I think that's what Java and Go use)

Reply:
Updated this to do 2 refreshes per 60s, but Java does 1/min. Should that be updated too? https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/blob/5cfa39e49a5362babf8412256f0081cda21c414f/core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java#L118

Reply:
Yeah, I think 2/min is probably an acceptable rate.